diff --git a/Modules/3.x/readline.c b/Modules/3.x/readline.c
--- a/Modules/3.x/readline.c
+++ b/Modules/3.x/readline.c
@@ -40,17 +40,13 @@ PyAPI_FUNC(char*) _Py_SetLocaleFromEnv(i
 #endif
 
 #undef WITH_EDITLINE /* We are building GNU readline */
-#ifdef WITH_EDITLINE
-#  include <editline/readline.h>
-#else
 /* GNU readline definitions */
-#  undef HAVE_CONFIG_H /* Else readline/chardefs.h includes strings.h */
+#undef HAVE_CONFIG_H /* Else readline/chardefs.h includes strings.h */
 /* Ensure that we get the intended local checkout of GNU readline
  * and not some system version (which could be libedit in disguise).
  */
 #  include "../../readline/readline.h"
 #  include "../../readline/history.h"
-#endif
 
 #ifdef HAVE_RL_COMPLETION_MATCHES
 #define completion_matches(x, y) \
